test/test_readwrite.jl

```diff
@@ -0,0 +1,34 @@
+using ITensors
+using ITensorInfiniteMPS
+using HDF5
```

Comment:
Try changing this to:
```julia
using ITensorInfiniteMPS.HDF5
```
This will use the "internal" version of HDF5 that will already be available in `ITensorInfiniteMPS`, so it doesn't need to try to separately install `HDF5`.

`using HDF5` can work, but you would have to add `HDF5` as a test dependency by activating the environment in `~/.julia/dev/ITensorInfiniteMPS/test` and doing `Pkg.add("HDF5")`.

Reply:
Thanks! Just did that and pushed a commit.